pages/_app.js

```diff
@@ -3,8 +3,10 @@ import GlobalStyle from "../styles";
 import data from "../data.json";
 import Layout from "@/components/Layout/Layout";
 import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
+  const [showForm, setShowForm] = useState(true);
```

Comment:
A better name for such a boolean is `isFormShown` to make clear that this is not a function.

Reply:
✅